modules/disabling-insights-advisor-recommendations.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="disabling-insights-advisor-recommendations_{context}"]
+= Disabling Insights Advisor recommendations
+
+You can disable specific recommendations impacting your clusters so they no longer appear in your reports. It is possible to disable a recommendation for a single cluster or all of your clusters. 
```

Comment:
Should we also mention that disabling the recommendation using `*Actions* -> *Disable recommendation*` disables it for current and "future" clusters?  I mention it because there is a way to disable clusters using a bulk selector which allows to select all existing clusters and disable the recommendation for all of them at once, but will not affect new clusters.

Reply:
Added a note